README.md

```diff
@@ -144,6 +144,8 @@ const config = {
   width: 900, // chart width
   height: 300, // chart height
   preview: false, // if true points wont be draggable
+  darkMode: false, // Whether the dark color scheme is to be used
+  backgroundColor: 'transparent', // Color to be used as bg, Use true for default color
```

Comment:
What would happen If I never set backgroundColor ?

Reply:
I've pushed a test to address this case